cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -233,7 +233,7 @@ def test_notifications_handler_dismiss(self):
             # delete nofications that are dismissed
             CourseRerunState.objects.get(id=rerun_state.id)
 
-        self.assertFalse(has_course_author_access(user2, rerun_course_key))
+        self.assertTrue(has_course_author_access(user2, rerun_course_key))
```

Comment:
I know you mentioned this before in conversation, but I've forgotten -- why did this behavior change?